src/app/userCodes/data/redisUserCodeStorage.js

```diff
@@ -39,39 +39,38 @@ const deleteCode = async (uid, client) => {
 };
 
 class RedisUserCodeStorage {
-  constructor(client) {
-    if (!client) {
-      if (!redisClient) {
-        redisClient = new Redis(config.adapter.params.redisurl);
-      }
+  constructor(redisClient) {
+    if (redisClient === null || redisClient === undefined) {
+      this.client = new Redis(config.userCodes.redisUrl);
     } else {
-      redisClient = client;
+      this.client = redisClient;
     }
   }
 
   async getUserPasswordResetCode(uid) {
     try {
-      return await find(uid, redisClient);
+      return await find(uid, this.client);
     } catch (e) {
-      throw e;
+      logger.error(e);
     }
   }
 
   async createUserPasswordResetCode(uid, clientId) {
     try {
-      return await createCode(uid, clientId, redisClient);
+      return await createCode(uid, clientId, this.client);
     } catch (e) {
-      throw e;
+      logger.error(e);
     }
   }
 
   async deleteUserPasswordResetCode(uid) {
     try {
-      return await deleteCode(uid, redisClient);
+      return await deleteCode(uid, this.client);
     } catch (e) {
-      throw e;
+      logger.error(e);
```

Comment:
Should we be throwing the error here? If not error it possibly not an appropriate log level